docs/README.md

```diff
@@ -395,6 +396,22 @@ Configures the git username and email to associate commits with the provided ide
 
 The two vars in the previous snippet are [workflow configuration variables](https://github.blog/changelog/2023-01-10-github-actions-support-for-configuration-variables-in-workflows/) that can be created at organization level and shared across different repositories.
 
+### delete-deployments
+
+Deletes the deployments created by a workflow run in the scope of the PR.
+
+```yaml
+  permissions:
+    deployments: write # This is required for deployment statuses management
+      - uses: Alfresco/alfresco-build-tools/.github/actions/delete-deployments@ref
```

Comment:
Please make it easier to copy/paste this: the omitted lines before this one are making that sample hard to navigate

Reply:
Done